image/scanner/scripts/import-additional-cas

```diff
@@ -1,8 +1,17 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 set -euo pipefail
 
-[ -d /usr/local/share/ca-certificates ] || exit 0
-[ "$(find /usr/local/share/ca-certificates -maxdepth 1 -name '*.crt' | wc -l)" -gt 0 ] || exit 0
-cp -L /usr/local/share/ca-certificates/* /etc/pki/ca-trust/source/anchors
+copy_existing () {
+    src=$1
+    if [ -d "$src" ] && [ "$(ls -A -I "..*" "$src")" ]; then
```

Comment:
1. Why is `-I "..*"` needed?
2. You can merge predicates
```suggestion
if [[ -d "$src" && -n "$(ls -A -I "..*" "$src")" ]]; then
```

Reply:
1. `-I "..*"` is there to ignore files like "..data" created by kubernetes volume mounting.
2. Yes, but why?

Reply:
2: Clarity. Each `[` spawns a command. In `[ something ] && [ something ]` there are at least two `[` commands.
Unlike `[`, `[[` is, I believe, a bash built-in that doesn't span extra command on its own. Plus `&&` inside of `[` or `[[` doesn't cause another command execution.
By _command_ I mean starting a process.

Reply:
Hm. I'm not sure how `[[a && -n b]]` is more clear than `[a] && [b]`.

Reply:
You can try `[[ a && b ]]`. Most likely it will work the same. Will it still be not clear enough?

Reply:
Regarding
> `-I "..*"` is there to ignore files like "..data" created by kubernetes volume mounting.

It is really not obvious from just looking at the code. I would recommend you adding this info as a comment in the code.